src/components/MenuButton/MenuButton.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { forwardRef, useCallback, useLayoutEffect, useMemo, useRef, useState } from "react";
 import cx from "classnames";
 import { camelCase } from "lodash-es";
+import { isForwardRef } from "react-is";
```

Comment:
I think it's worth doing what you did before
have a util that exports the logic

Reply:
wdyt about creating a new `lib` under `utils` or as a sibling dir of util? and let's migrate slowly all of our imports from libraries like lodash, etc. so we can import from `lib` instead of importing from libraries all over our app.
we can start with your new `isForwardRef` to start pushing it

Reply:
a suggestion:

. lib
|----- react-is.ts (that includes whatever utils we'll need to export)
|----- loadsh.ts (same) (not for this PR)
|----- index.ts (?) (to import from /lib whatever needed, not necessarily from /lib/react-is)

Reply:
What for? If it's one function which can be imported directly...

Reply:
it's a better practice for cases you want to change a lib and keep the logic, or log stuff every time you want to call some function
it's ok to leave it like that for now, but it's just a better practice